_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+
+```{r echo=FALSE, fig.cap="Comparison of blogdown vs distill styled using R-Ladies Google slide template."}
+knitr::include_graphics("img/compare_blogdown_distill.svg")
+```
+
+
+I also created five R-Ladies styled code gifs with xaringan and flipbookr - methods and code are in this [blog post](https://www.pipinghotdata.com/posts/2021-03-08-r-ladies-styled-code-gifs-with-xaringan-and-flipbookr/){target="_blank"}. Here is an example code gif:
+
+```{r echo=FALSE, fig.cap="Example R-Ladies styled code gif."}
+knitr::include_graphics("img/demo-walrus-flipbook-quality.gif")
+```
+
+
+
+
+# First and last tweets
+
+The introduction and farewell tweets as a curator are important as this is when you actually tell people your name or personal twitter handle. I have seen 
+many excellent curators fall short on engagement (likes and re-tweets) with their first and last tweets, resulting in less than merited recognition for the wonderful curator. 
+
+To address this, I aimed to create content-rich [first](https://twitter.com/WeAreRLadies/status/1361286341317779456){target="_blank"} and [last](https://twitter.com/WeAreRLadies/status/1363286037846511616){target="_blank"} tweets to give users more motivation to like or re-tweet, and I also connected the content with links to my blog so that users could easily learn more about me.
+
+# TweetDeck
+
+When you serve as a curator, you will be tweeting from [TweetDeck](https://tweetdeck.twitter.com/){target="_blank"}, and it is hard to separate curator experience from the technology. Tweeting from TweetDeck can be overwhelming compared to the standard Twitter interface.
+
+Moreover, there were limitations to the platform that added challenges to curating, which included:
+
+1. There was no `+` enabled to easily create threads (I had to send a tweet and then comment on the tweet, and it was initially hard to ensure the thread appeared in correct order).
+
+2. Consequently, I could not draft and save threads in TweetDeck to send later.
+
+3. I could not send polls from the curator account on TweetDeck; polls were sent from my personal account and then re-tweeted from the curator account.
+
+4. Depending on the content already in the send tweet interface, sometimes other options in TweetDeck would disappear, like the emoji, gifs, and upload image buttons. I kept [emojipedia](https://emojipedia.org/){target="_blank"} open to easily copy and paste emojis into my tweets, and it took trial and error to get everything I wanted in a single tweet.
+
+5. When uploading local content, you can add [descriptions](https://help.twitter.com/en/using-twitter/picture-descriptions){target="_blank"} to both gifs and images in the regular Twitter interface to create inclusive content for community members that use assistive reading technology; however, in TweetDeck, descriptions were enabled for images but not gifs.
```

Comment:
I find this section super important! For me another problem is that sometimes TweetDeck jumped from the @WeAreRLadies to my personal account so I replied or tweeted few times from my personal account and I had to delete and try again. I really found the TweetDeck limiting! But again, I am not a heavy social media user, so it might be me!

Reply:
good point - yes, that happend to me as well!

Reply:
updated tweetdeck section here https://github.com/shannonpileggi/pipinghotdata_distill/pull/2/commits/72a4b55b6c54e4c5e3ab471ed7f8c3a1251fdb65